src/components/Overview.tsx

```diff
@@ -1,47 +1,73 @@
 import { useEffect, useState } from "react";
 import useFetchArtworks from "../hooks/useFetchArtworks";
 import GalleryGrid from "./GalleryGrid";
+import FiltersModal from "./FiltersModal";
+import "../modal.css";
 
 function Overview() {
-  const [isOnView, setIsOnView] = useState(
-    () => localStorage.getItem("isOnView") === "true"
-  );
-  const [isPublicDomain, setIsPublicDomain] = useState(
-    () => localStorage.getItem("isPublicDomain") === "true"
-  );
-  const [isHiddenGem, setIsHiddenGem] = useState(
-    () => localStorage.getItem("isHiddenGem") === "true"
+  const [localFilters, setLocalFilters] = useState(() => ({
+    isOnView: localStorage.getItem("isOnView") === "true",
+    isPublicDomain: localStorage.getItem("isPublicDomain") === "true",
+    isHiddenGem: localStorage.getItem("isHiddenGem") === "true",
+    searchQuery: localStorage.getItem("searchQuery") || "",
+  }));
+
+  const [filters, setFilters] = useState(localFilters);
+
+  const isFiltersActive =
+    filters.isOnView ||
+    filters.isPublicDomain ||
+    filters.isHiddenGem ||
+    filters.searchQuery !== "";
+
+  const handleSearch = (e: React.FormEvent) => {
+    e.preventDefault();
```

Comment:
Actually, preventing the default behavior shouldn't be the concern of the `<Overview>` component. This should still be handled in the modal